src/features/canvas/CanvasIntegration.ts

```diff
@@ -4,8 +4,8 @@ const CANVAS_API = 'https://sit.instructure.com/api/v1/';
 function getUpcomingAssignments() {
     chrome.storage.local.get('tokens', tokens => {
         // If you want to test this, uncomment the code below with your token
-        // tokens.canvas = '<YOUR TOKEN HERE>';
-        if (tokens.canvas !== '') {
+        // tokens.canvas = '<YOUR TOKEN>';
+        if (tokens.canvas !== '' && tokens.canvas !== undefined) {
```

Comment:
Since both of those are falsy
```suggestion
        if (!tokens.canvas) {
```